### DISCUSSION TITLE
Change beta version one step plus stable version (ex 1.3.2 beta 1.3.1 stable)

### DISCUSSION THREAD

**Discussion by bovirus** [rustdesk/rustdesk#9431] [General]
### Bug Description

@rustdesk

When you release a new stable version please change new beta version one step plus the stable version.

Now stable version is 1.3.1 and beta is 1.3.1.

### How to Reproduce

Check file download

### Expected Behavior

Beta version should be always one step plus then stable version.

### Operating system(s) on local side and remote side

Windows 10 22 H2 64 bit Italian

### RustDesk Version(s) on local side and remote side

1.3.1

### Screenshots

No screenshot

### Additional Context

No other info

**Comment by xlionjuan** [comment]
They will, but may not very quickly.
